csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -2278,6 +2283,69 @@ The Weakness name (`name`) has value type `string` with 1 or more characters and
 
 Discovery date (`discovery_date`) of value type `string` with format `date-time` holds the date and time the vulnerability was originally discovered.
 
+#### 3.2.3.XYZ Vulnerabilities Property - Flags
+
+List of flags (`flags`) of value type `array` with 1 or more items of type `object` contains a list of machine readable flags.
+
+```
+    "flags": {
+      // ...
+      "items": {
+        // ...
+      }
+    },
+```
+
+Every Flag item of value type `object` with the mandatory property Label (`label`) contains product specific information in regard to this vulnerability as a single machine readable flag.
+For example, this could be a machine readable justification code why a product is not affected.
+
+> These flags MAY be used to decide in an automated fashion which actions to take.
```

Comment:
The shouting `may` normally alerts the `other` partner in sender/receiver, producer/consumer, or server/client protocols to deal with complications arising from an alternate reality. Here, we intend to suggest a use outside of the `parsing` arena.

Something like the following wording might be closer to the intended message:
> These flags enable the receiving party to automate the selection of actions to take.

Reply:
Sorry - did that again. :see_no_evil:
Thank you for the suggestion. Fixed in [103ff4b](https://github.com/oasis-tcs/csaf/pull/460/commits/103ff4bba8f934d5b151f6e06f3b870078f688ba).